app/src/main/java/com/bignerdranch/android/androidacademy/ActorViewHolder.kt

```diff
@@ -0,0 +1,17 @@
+package com.bignerdranch.android.androidacademy
+
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+
+class ActorViewHolder(itemView: View): RecyclerView.ViewHolder(itemView) {
+    private val avatar = itemView.findViewById<ImageView>(R.id.iv_avatar)
+    private val name = itemView.findViewById<TextView>(R.id.tv_name)
+
+    fun onBind(actor: Actor){
+        avatar!!.setImageResource(actor.img)
```

Comment:
why you use !!(assertion)
it's redundant

Reply:
also by mistake. I'll take this kind of mistakes into consideration next time